### DISCUSSION TITLE
9704 Gen 4 Modules - Impossible To Get

### DISCUSSION THREAD

**Discussion by bcramer-mcc** [jgyates/genmon#1136] [General]
Hello,

So I've been trying for months to get these 9704 Gen 4 Modules. It seems like they are impossible to get. 

I also got a lashing about going this route:

Brandon

We have relayed your information below to another rep with Rochester and they did reply with the response below.

We cannot support this as the below approach is in violation of our intellectual property.

[https://github.com/jgyates/genmon/wiki/Appendix-L-Adding-a-Propane-Fuel-Gauge-to-Genmon](https://us-west-2.protection.sophos.com/?d=github.com&u=aHR0cHM6Ly9naXRodWIuY29tL2pneWF0ZXMvZ2VubW9uL3dpa2kvQXBwZW5kaXgtTC1BZGRpbmctYS1Qcm9wYW5lLUZ1ZWwtR2F1Z2UtdG8tR2VubW9u&p=m&i=NjYzZTMxMWRiZGJmMzcxN2U2OTg0NWUw&t=ZFJtbjNWVlBCM0g4RkhJV0hFSnBIejltejdPVElBVVQzcjVlRjBvTm01Yz0=&h=45734d6afb444cf6a2344499c6b1e32e&s=AVNPUEhUT0NFTkNSWVBUSVa16u7_JpUo6sx0segdUGBEvn8sMVJ5RRkeUoGS4_khRw)

This approach also is not hazardous location approved and therefore the user would assume all risk and liability should there be an explosion.  

You should send this customer to a telemetry provider such as Otodata or Independent Technologies (Anova) as they could provide a turnkey solution for his application that is approved for use in a propane application.

End of Lashing...

Should I go with DIY Tank Method 2 then? I'm not crazy about going this route but the Mopeka sensors just don't work consistently. I assume if I was to do this route that I could still use my Raspberry Pi 4's? Also, I believe the length wiring from the R3D Propane Gauge Module to my Genmon would need to be 50 to 100 feet. Is that going to be an issue?

Thanks,
Brandon

I have been using my personal github account. This was under bcrazycramer. I created my own business github account.

**Comment by jgyates** [comment]
I am not really surprised. Some sales people get nervous when they think you are going to take apart the products you buy. I would just ask for the part (if they have it) and not just tell them you are replacing an existing part that quit working.

The generac propane tank monitor uses wifi (formally owned by tankutility.com). This has it's limits as well.

As far as the limit for the R3D cable length,. I do not know as that has a magnetic field sensor. You would need to consult the data sheet for the TLE5501 or someone else can chime in with their knowledge. Use @curtis1757  may know the answer.

**Reply by curtis1757** [reply]
The length of my propane sensor cable is about 18 feet with no problems. See my DIY sensor article for the sensor type I'm using.
Curtis

**Comment by jrtaylor71** [comment]
Might help others looking for a sensor.

https://www.ebay.com/itm/116316549800?_skw=Rochester+Sensors+9701&itmmeta=01JCY5WYJ624H5SFKC8BA8HWS1&hash=item1b1501b2a8:g:AJIAAOSwYCtm2l2l&itmprp=enc%3AAQAJAAAA8HoV3kP08IDx%2BKZ9MfhVJKmvzMmH2c0cDJLuqeDTb%2B268YlMulnyAA346VOxc17FtDSCyHaaQMAeMOKzGwco5TSw6uhEHLqjpvmxjm%2FwLaeS1nXP3FMfSIYazIugRuQZ8pO0t03X0XtThrZH7xTS3eYKf2l%2Fl2fxhZikFE2%2ByfzP3qutn1EytoVADVKkXC0g9wO8awACayUIYSYoZWJ4i84Mr4PdWgqy4gZLZLEYmGKWDWXj5Uq%2FReezeCOdRF9mJvlnQVmJtl7e%2BWuZZ3oZW36DifaKolCnPvUNY6Tt9gbSaA0W%2B0iEvchoNrBiQM8QBQ%3D%3D%7Ctkp%3ABk9SR5jp88XnZA

**Comment by rtvanhook** [comment]
If you gotta have the angled version (because the ebay one looks great for the straight model) and  can handle highway robbery and outrageous shipping prices, search for part number "SQU 130171".  I just paid $73 for the detector and $40 for UPS shipping.  I only wish I were kidding.

Another comment...per the data sheet 

[DS-02011 Hall Effect Sensor Data Sheet.pdf](https://github.com/user-attachments/files/18390902/DS-02011.Hall.Effect.Sensor.Data.Sheet.pdf)

this is a very high-impedance (> 500 k ohms) DC analog sensor.  The few tenths of ohms a longer wire might add will make no difference.  Ohm's law still applies.  A low voltage source with a high-impedance load will have essentially zero current within a few milliseconds of applying power. 

A couple other things:

1.    The sender measures magnetic fields.  To make it more leak-tight (it is, after all, measuring an explosive gas...), there's no direct connection between the float and the dial.  Instead, a float inside the tank spins a permanent magnet right behind a thin aluminum plate just under the dial.  My point:  the sender isn't reading the dial.  Both the dial and sender are reading the orientation of the same spinning permanent magnet.  This explains some of the nonsensical readings that some folks have gotten bench-testing.  Because they're probably not bench-testing in the same magnetic environment as on top of the tank.

2.  For the same reason, I'm skeptical about some of the online folks who have jury-rigged various angle-sensor chips.  I would think that precise orientation is needed, and that would be hard for me to get.  So I bit the bullet.  Search Google for that part number.

**Reply by bcrazycramer** [reply]
Are you searching eBay for that part number SQU 130171?

**Reply by rtvanhook** [reply]
No.  Google.  I don't want to link to it, because it's the only source I could find and..if they're not supposed to sell to the general public...I don't want them to figure that out.

**Reply by gm58** [reply]
> Are you searching eBay for that part number SQU 130171?

@bcrazycramer
Is this what your looking for?

https://kscdirect.com/item/SQU%2B130171/SQUIBB%2BTAYLOR%2BCOMPANY_9704-RD106M%2BANG%2BR3D%2BGEN4%2BMODULE%2BW%252F6%2527%2BCBL#extdesc
